server/src/main/java/org/apache/iotdb/db/metadata/MManager.java

```diff
@@ -1843,7 +1889,7 @@ private void internalCreateTimeseries(PartialPath path, TSDataType dataType)
           getDefaultEncoding(dataType),
           TSFileDescriptor.getInstance().getConfig().getCompressor(),
           Collections.emptyMap());
-    } catch (PathAlreadyExistException | AliasAlreadyExistException e) {
+    } catch (PathAlreadyExistException | AliasAlreadyExistException e) {   // TODO: 2021/1/7 这个exception不会抛出来
```

Comment:
do not leave Chinese in source codes.

Reply:
Opps, My fault... I'll be very care of not pushing such Chinese comments in the next push that solves all the reasonable problems.